self_tests/measure-cyclomatic-complexity.sh

```diff
@@ -12,4 +12,8 @@ function prepare_venv() {
 
 [ "$NOVENV" == "1" ] || prepare_venv || exit 1
```

Comment:
Not related to this PR but similarly to https://github.com/fabric8io/fabric8-test/pull/875#discussion_r202946434, can we use native Python 3.4 venv instead of `virtualenv`?

Reply:
yeah why not :) We just need to make sure it is available on RHEL/CentOS, so we'll be able to run these scripts on CI.